internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -315,6 +315,64 @@ func resourceMsSqlVirtualMachine() *pluginsdk.Resource {
 				ValidateFunc: validate.SqlVirtualMachineLoginUserName,
 			},
 
+			"sql_instance_setting": {
```

Comment:
setting seems redundant?
```suggestion
			"sql_instance": {
```

Reply:
Have removed `_setting`.